holoviews/plotting/bokeh/callbacks.py

```diff
@@ -53,35 +55,55 @@ def attributes_js(attributes, handles):
 class Callback(object):
     """
     Provides a baseclass to define callbacks, which return data from
-    bokeh models such as the plot ranges or various tools. The callback
+    bokeh model callbacks, events and attribute changes. The callback
     then makes this data available to any streams attached to it.
 
     The definition of a callback consists of a number of components:
 
-    * handles    :  The handles define which plotting handles the
-                    callback will be attached on, e.g. this could be
-                    the x_range, y_range, a plotting tool or any other
-                    bokeh object that allows callbacks.
+    * models     :  Defines which bokeh models the callback will be
+                    attached on referencing the model by its key in
+                    the plots handles, e.g. this could be the x_range,
+                    y_range, plot, a plotting tool or any other
+                    bokeh mode.
 
-    * attributes :  The attributes define which attributes to send
+    * extra_models: Any additional models available in handles which
+                    should be made available in the namespace of the
+                    objects.
+
+    * attributes  : The attributes define which attributes to send
                     back to Python. They are defined as a dictionary
                     mapping between the name under which the variable
                     is made available to Python and the specification
                     of the attribute. The specification should start
                     with the variable name that is to be accessed and
-                    the location of the attribute separated by periods.
-                    All plotting handles such as tools, the x_range,
-                    y_range and (data)source can be addressed in this
+                    the location of the attribute separated by
+                    periods.  All models defined by the models and
+                    extra_models attributes can be addressed in this
                     way, e.g. to get the start of the x_range as 'x'
                     you can supply {'x': 'x_range.attributes.start'}.
                     Additionally certain handles additionally make the
                     cb_data and cb_obj variables available containing
                     additional information about the event.
 
+    * skip        : Conditions when the Callback should be skipped
+                    specified as a list of valid JS expressions, which
+                    can reference models requested by the callback,
+                    e.g. ['pan.attributes.active'] would skip the
+                    callback if the pan tool is active.
```

Comment:
If the models references in the skip conditions need to be in ``extra_models`` it would be helpful to say so.